perun-base/src/main/resources/perun-base.xml

```diff
@@ -90,7 +90,7 @@
 					perunRegistrar, perunSynchronizer, perunCabinet, perunNotifications, perunRpc, perunLdapc,
 					perunLdapInitializer
 				</prop>
-				<prop key="perun.db.type">hsqldb</prop>
+				<prop key="perun.db.type">postgresql</prop>
```

Comment:
We should probably re-think necessity of this property/config option, since Perun supports only one DB backend now.

Reply:
Not removed at this PR.